modules/webhook/error.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package webhook
```

Comment:
The model errors can't reside in the models package?

Reply:
Well, they could, but I am not too fond of having more things in models than necessary.
For me, the more things we can outsource to modules, the better…
But yes, if you absolutely want to, we can move it back.

Reply:
For me they belong directly to the model like we do it everywhere else.

Reply:
What's the opinion of others on this?
@lunny, @6543?

Reply:
The error struct should be stayed near which it produced. I think revert this error change is better. These two errors are returned first in `models/hooktask.go`, so I prefer to move them back to that file.